src/ResourceManager/Common/Commands.Common.Strategies/IState.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Microsoft.Azure.Commands.Common.Strategies
+{
+    /// <summary>
+    /// An Azure state which is a dictionary of models.
+    /// </summary>
+    public interface IState
+    {
+        Model GetOrNull<Model>(IResourceBaseConfig<Model> config)
+            where Model : class;
+
+        object GetOrNullUntyped(IResourceBaseConfig config);
+    }
+
+    /*
```

Comment:
@sergey-shandar please remove this commented code

Reply:
Done.